### PR TITLE
Story1

### PR DESCRIPTION
**Pull request by sassiek**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?php
+include_once 'db.php';
+$db = getSupercarData();
+
```
Comment (nealioa) [inline index.php]: Do you need a blank line?

File: db.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+function getSupercarData()
+{
+    $db = connectToDB();
+    $query = $db->prepare("SELECT `make`, `model`, `bhp`, `top_speed_mph` FROM `supercars`");
+    $query->execute();
+    $result = $query->fetchAll();
+    return $result;
+}
+
+function displaySupercarData(array $supercars):string
```
Comment (nealioa) [inline db.php]: Put a space after the colon (`: string`).

File: index.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+<?php
+echo displaySupercarData($db);
+?>
+</body>
+</html>
+
```
Comment (nealioa) [inline index.php]: Delete the blank lines below here.

File: supercarcollectorapp/db.php [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+{
+   public function testDisplaySupercarData_success()
+    {
+        $input = [
+            [
+                'make' => 'Aston Martin',
+                'model' => 'One-77',
+                'bhp' => '750',
+                'top_speed_mph' => '220'
+                ]
```
Comment (nealioa) [inline supercarcollectorapp/db.php]: Put the closing square bracket in line with the opening one (line 11).

**Review by nealioa — CHANGES_REQUESTED** [review]

**Review by mporam — CHANGES_REQUESTED** [review]
change db.php into functions.php

File: functions.php [hunk 5]
```diff
@@ -1,7 +1,7 @@
 <?php
 function connectToDB()
 {
-    $db = new PDO('mysql:host=db;dbname=supercarcollectorapp', 'root', 'password');
+    $db = new PDO('mysql:host=functions;dbname=supercarcollectorapp', 'root', 'password');
```
Comment (mporam) [inline functions.php]: you havent tested this have you! this wont work, the host has to be `db` - it has no relation to the name of the file, `db` is just what the docker config says it has to be

**Review by mporam — CHANGES_REQUESTED** [review]

**Review by mporam — APPROVED** [review]

**Pull request merged**